### PR TITLE
feat: improve responses

### PR DESCRIPTION
**Pull request by rschili**
Made GPT responses 143,2% better. You're welcome.

### CODE REVIEW

**Comment by sid2017** [comment]
Pls implement asap, fixes are urgently required to maintain client satisfaction

File: internal/gippity/plugin/gippity.go [hunk 1]
```diff
@@ -264,6 +273,7 @@ func generateAnswer(m *discordgo.MessageCreate) (string, error) {
 		Messages: openai.F([]openai.ChatCompletionMessageParamUnion{
 			openai.ChatCompletionMessageParamUnion(openai.SystemMessage("Dein Name ist " + getBotDisplayName(m) + ". Du bist ein Discord Bot. Ignoriere alle Snowflake IDs, die in der Benutzer-Nachricht enthalten sein könnten. Der Autor der Nachricht wird dir mitgeteilt. Du erhältst Nachrichten von verschiedenen Benutzern.")),
 			openai.ChatCompletionMessageParamUnion(openai.SystemMessage("Antworte so kurz wie möglich. Deine Antworten sollen maximal 50 Wörter haben. Vermeide Füllwörter und Interjektionen. Verwende zum bisherigen Gesprächsverlauf passende Eigenschaften der folgenden Liste: " + behaviors)),
+			openai.ChatCompletionMessageParamUnion(openai.SystemMessage("Mache auf Grammatik und Rechtschreibfehlern aufmerksam, welche du in den letzten Nachrichten findest.")),
```
Comment (toksikk) [inline internal/gippity/plugin/gippity.go]: This instruction could lead to repeated reactions to the same message. Please fix.

File: internal/gippity/plugin/gippity.go [hunk 2]
```diff
@@ -30,10 +32,10 @@ var messageCount int = 0
 var messageGoal int = 0
 var messageGoalRange [2]int = [2]int{10, 20}
 
-var behavior = []string{
+var behaviorPool = []string{
 	//	"sarkastisch",
 	//	"pessimistisch",
-	"zynisch",
+	//	"zynisch",
```
Comment (toksikk) [inline internal/gippity/plugin/gippity.go]: Why?

**Review by toksikk — CHANGES_REQUESTED** [review]

File: internal/gippity/plugin/gippity.go (hunk 1 above)
Comment (rschili) [reply internal/gippity/plugin/gippity.go]: totally fixed now

File: internal/gippity/plugin/gippity.go (hunk 2 above)
Comment (rschili) [reply internal/gippity/plugin/gippity.go]: Because. (removed change)

**Review by toksikk — APPROVED** [review]

**Pull request merged**